index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
+</head>
+
+<body>
+
+  <header>
+    <nav>
+      <a href="#"><img src="images/MyLogo.png" alt="Logo"></a>
+      <a href="#"> <img src="images/hamb.png" alt="hamb" class="humb"> </a>
+    </nav>
+  </header>
+  <section class="intro">
+    <div class="head">
+      <h1>
+        I’m Lumato
+        Glad to see you!
+      </h1>
+    </div>
+
```

Comment:
**OPTIONAL**
Please keep in mind that we're only adding blank lines and indentations for good reasons such as separating two logical pieces of code or two large sections apart. This helps to make the code cleaner and the website more optimized.

> You can read about this point and other useful ones [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).